fence/src/main/java/io/kidsfirst/fence/Utils.java

```diff
@@ -69,17 +70,15 @@ private static Boolean isKmsEnabled() {
     }
 
     public static void persistTokens(String userid_in_fence, String userid_in_ego, String access_token, String refresh_token) {
-
-        getDynamoDB().batchWriteItem(
-                new TableWriteItems(getTokenTableName())
-                        .withItemsToPut(
-                                new Item()
-                                        .withPrimaryKey(FIELD_NAME_OF_USER_ID_IN_EGO, userid_in_ego)
-                                        .withString(FIELD_NAME_OF_USER_ID_IN_FENCE, userid_in_fence)
-                                        .withString(FIELD_NAME_OF_ACCESS_TOKEN, encrypt(access_token))
-                                        .withString(FIELD_NAME_OF_REFRESH_TOKEN, encrypt(refresh_token))
-                        )
-        );
+        FenceToken token = new FenceToken(userid_in_ego);
+        token.setFenceUserId(userid_in_fence);
+        token.setAccessToken(encrypt(access_token));
+        token.setRefreshToken(encrypt(refresh_token));
```

Comment:
I have a huge temptation to return the `FenceToken` instance so this can be chained like jquery, but my brain is fried on Java

Reply:
@hlminh2000 , that is what the builder pattern is for!

Reply:
If you use `lombok`'s `@Builder` annotation on the FenceToken class. it will provide a builder for you. 

So you could invoke it:
```
val token = builder().fenceUserId(userid_in_fence).accessToken(access_token).refreshToken(refresh_token).build();
```

Reply:
The whole idea would be avoid mutating any value, hence a builder rather than chaining.

Reply:
Ah gotcha! That makes a lot of sense :) thanks @andricDu for the review